scripts/install-arch-aarch64.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+# This script installs Klipper on an aarch64 Arch Linux system
+
+PYTHONDIR="${HOME}/klippy-env"
+SYSTEMDDIR="/etc/systemd/system"
+CONFIGDIR="${HOME}/klipper_config"
+KLIPPER_USER=$USER
+KLIPPER_GROUP=$KLIPPER_USER
+
+# Step 1: Install Paru to handle AUR builds
+install_paru()
+{
+    # Packages needed to install Paru
+    PKGLIST="git base-devel"
+
+    # Install desired packages
+     report_status "Installing Paru dependencies..."
+
+     sudo pacman -S --needed ${PKGLIST}
+
+     # Clone paru-bin
+     git clone https://aur.archlinux.org/paru-bin.git ~/paru-bin
+
+     # Install Paru
+     report_status "Installing Paru..."
+     cd ~/paru-bin
+     makepkg -si
+
+     # Remove git repo
+     cd ~
+     sudo rm -rf paru-bin
+}
+
+# Step 2: Install Packages
+install_packages()
+{
+    # Packages for python cffi
+    PKGLIST="python2-virtualenv libffi"
+    # kconfig requirements
+    PKGLIST="${PKGLIST} ncurses"
+    # hub-ctrl
+    PKGLIST="${PKGLIST} libusb"
+    # AVR chip installation and building
+    PKGLIST="${PKGLIST} avrdude avr-gcc avr-binutils avr-libc"
+
+    # Install desired packages
+    report_status "Installing packages..."
+    paru -S --needed ${PKGLIST}
+     
+    # Install ARM build package. PKGBUILD recompresses it, so this takes a *LONG* time
+    # even on a Pi 4
+    report_status "Installing arm-none-eabi-gcc. This will take some time..."
+    paru -S --skip-review gcc-arm-none-eabi-bin
```

Comment:
for me it doesn't work. I think --skip-review should be --skipreview.

Reply:
Yup, that's what I get for changing something last minute. Should be fixed now.